docroot/sites/registrar.uiowa.edu/modules/registrar_core/js/academic_calendar.js

```diff
@@ -9,59 +9,75 @@
         let uniqueSessions = new Set();
 
         // Cache objects for frequently used elements.
-        const $form = $('#academic-calendar-filter-form', context);
-        const $groupByMonthCheckbox = $('#group-by-month', context);
-        const $showPreviousEventsCheckbox = $('#show-previous-events', context);
+        const form = context.querySelector('#academic-calendar-filter-form');
+        const showPreviousEventsCheckbox = context.querySelector('#show-previous-events');
+        const categoryChosen = context.getElementById('edit_category_chosen');
```

Comment:
Curious about the use of underscores instead of dashes (like in the next item). I understand this is finding an ID that is defined somewhere else, but maybe we should change that value to use dashes as well?

Reply:
Chosen adds that element I believe, and the others were pre defined. I am open to the change if it is desired.